apps/confidential/confidential.go

```diff
@@ -283,6 +283,8 @@ type Options struct {
 
 	// Instructs MSAL Go to use an Azure regional token service with sepcified AzureRegion.
 	AzureRegion string
+	//Validate Authority
+	ValidateAuthority bool
```

Comment:
We should not use the term `ValidateAuthority`, this is something that we are trying to deprecate. Consider using `IsInstanceDiscoveryEnabled`